### DISCUSSION TITLE
Twitter Sign In Problem

### DISCUSSION THREAD

**Discussion by floheitzmann** [0maru/twitter_login#77] [Q&A]
Hello,
I have a problem with the sign-in process. It works fine till the redirect.
![Screenshot 2022-01-29 at 14 10 15](https://user-images.githubusercontent.com/7592093/151662246-669bc0f7-f03f-4d42-b353-03f0eb35b4ef.png)

How can I fix this problem? Does this happens only on the iOS Simulator?

**Comment by 0maru** [comment]
@scarpex 
This plugin cannot use firebase url.
Please use deeplink.
